base/applications/cmdutils/more/lang/ro-RO.rc

```diff
@@ -1,60 +1,64 @@
-    command                 A command whose output is to be displayed.\n\
+    [disc:][cale]numele fișierului  Un fișier al cărui conținut urmează să fie afișat.\n\
+    comandă                         O comandă a cărei ieșire urmează să fie afișată.\n\
 \n\
-When Extended Features are enabled, the following form is supported:\n\
-MORE [/E] [options] [files]\n\
+Atunci când opțiunea Caracteristici extinse e activată, următoarea formă este
+acceptată:\n\
+MORE [/E] [opțiuni] [fișiere]\n\
 \n\
-Options:\n\
-    /E       Enable Extended Features.\n\
-    The default state of the Extended Features is controlled by the default\n\
-    registry settings for CMD.EXE. For more details on how to change these\n\
-    settings, see the section ""Command Extensions"" in CMD.EXE help.\n\
+Opțiuni:\n\
+    /E       Activează opțiunea Caracteristici extinse.\n\
+    Stadiul implicit al Caracteristicilor extinse e controlat de\n\
+    configurațiile implicite de registru pentru programul CMD.EXE. Pentru mai\n\
+    multe detalii legate de cum pot fi schimbate aceste configurații, vedeți\n\
+    secțiunea ""Extensii de comenzi"" în secțiunea de ajutor a programului\n\
+    CMD.EXE.
 \n\
-    When Extended Features are enabled, the following options are available:\n\
-    /C       Clear the screen before displaying a page.\n\
-    /P       Expand Form-Feed characters.\n\
-    /S       Shrink successive blank lines to a single line.\n\
-    /Tn      Expand tabs to n spaces (default: 8).\n\
+Atunci când opțiunea Caracteristici extinse e activată, sunt disponibile\n\
+următoarele opțiuni:
+    /C       Curăță ecranul înainte de a se afișa o pagină.\n\
+    /P       Extindeți caracterele Form-Feed (de alimentare de formulare).\n\
+    /S       Reduceți liniile goale succesive la o singură linie.\n\
+    /Tn      Extinde filele la n spații (implicit: 8).\n\
 \n\
-    These options can be specified in the MORE environment variable.\n\
+    Aceste opțiuni pot fi specificate în variabila de mediu MORE.\n\
 \n\
-    +n       Start displaying the first file at line n.\n\
-    files    List of files to display, separated by blanks.\n\
+    +n       Începe afișarea primului fișier la linia n.\n\
+    fișiere  Listează fișierelor de afișat, separate prin spații libere.\n\
 \n\
-When Extended Features are enabled, the following commands are available\n\
-at the ""-- Continue --"" prompt:\n\
+Atunci când opțiunea Caracteristici extinse e activată, sunt disponibile\n\
+următoarele comenzi la panoul ""-- Continuă --"":\n\
 \n\
-    P n      Display the next n lines.\n\
-    S n      Skip the next n lines.\n\
-    F        Display the next file.\n\
-    Q        Quit.\n\
-    =        Show the line number.\n\
-    ?        Show the help line.\n\
-    <Space>  Display the next page.\n\
-    <Enter>  Display the next line.\n"
+    P n      Afișează următoarele n linii.\n\
+    S n      Omite următoarele n linii.\n\
+    F        Afișează următorul fișier.\n\
+    Q        Ieși.\n\
```

Comment:
The texts in the above lines are describing the parameters, and [indicative mood](https://en.wikipedia.org/wiki/Realis_mood) should be (and is) used, but for "quit" you (erroneously) used a translation in [imperative mood](https://en.wikipedia.org/wiki/Imperative_mood). Remember that the text does not give orders, it only explains something.